components/AppLayout/AppLayout.js

```diff
@@ -111,14 +114,21 @@ function AppLayout({ children, container = true }) {
 
   return (
     <Fragment>
-      <AppHeader
-        user={data?.GetUser}
-        showProgress={isRouteChanging}
-        onMenuButtonClick={toggleSidebar}
-        onLoginModalOpen={openLoginModal}
-        onLogout={logout}
-        onNameChange={handleNameChange}
-      />
+      {pathname === '/' ? (
```

Comment:
I prefer not using `AppLayout` in `src/index` at all, instead of adding more logic to `AppLayout` to support the landing page.

`AppLayout` is designed for application pages with header. It seems that the only common part between application pages and the landing page is the app footer and the login dialog. If that is the case, I suggest using `AppLayout/AppFooter` and `AppLayout/LoginModal` in `src/index` directly. Also, `LandingPageHeader` can be isolated from `AppLayout/AppHeader` and move to under `LandingPage` directory; since it's only used in `LandingPage`, it should be more straightforward to host it there.